doc/classes/Viewport.xml

```diff
@@ -33,6 +33,18 @@
 				Returns the first valid [World3D] for this viewport, searching the [member world_3d] property of itself and any Viewport ancestor.
 			</description>
 		</method>
+		<method name="get_audio_listener_2d" qualifiers="const">
+			<return type="AudioListener2D" />
+			<description>
+				Returns the currently active 2D audio listener. Returns [code]null[/code] if there are no active 2D audio listeners, in which case the active 2D camera will be treated as listener.
```

Comment:
2D uses viewport transform. Unlike 3D, you can hear sounds without any Camera2D in the scene.